hw/ip/otbn/dv/model/otbn_model.h

```diff
@@ -158,6 +161,8 @@ class OtbnModel {
 
   OtbnMemUtil mem_util_;
   std::string design_scope_;
+
+  bool disable_stack_check_flag = false;
```

Comment:
```suggestion
  bool stack_check_enabled_ = true;
```
I find it easier to reason and keep track of things being enabled than disabled. (For instance, "do something when enabled" seems more natural to me than "do something when not disabled".) Also, private member variables commonly end with an underscore.